packages/interaction-kit/src/definitions/application-commands.ts

```diff
@@ -127,18 +127,20 @@ export type ApplicationCommandInteractionDataOption = {
 // HACK: This is to fix typechecking
 export type OptionType = unknown;
 
-// https://discord.com/developers/docs/interactions/slash-commands#interaction-response
+// https://discord.com/developers/docs/interactions/slash-commands#interaction-response-object-interaction-response-structure
 // TODO: Check?
 export type InteractionResponse = {
 	type: InteractionCallbackType;
 	data?: InteractionApplicationCommandCallbackData;
 };
 
-// https://discord.com/developers/docs/interactions/slash-commands#interaction-response-interactioncallbacktype
+// https://discord.com/developers/docs/interactions/slash-commands#interaction-response-object-interaction-callback-type
 export enum InteractionCallbackType {
 	PONG = 1,
 	CHANNEL_MESSAGE_WITH_SOURCE = 4,
 	DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE = 5,
+    	DEFERRED_UPDATE_MESSAGE = 6,
```

Comment:
Hmmm do I need to change some of my methods to return this?

Reply:
To be honest, I have no idea - I didn't check where these are used, I just added them.